src/js/outreach/views/opt-in_views.js

```diff
@@ -79,8 +113,17 @@ const OptInView = View.extend({
   onChangeLastName() {
     this.model.set({ last_name: trim(this.ui.lastName.val()) });
   },
-  onChangeBirthDate() {
-    this.model.set({ birth_date: trim(this.ui.birthDate.val()) });
+  onChangeDobInput() {
+    const dobMonth = this.ui.dobMonth.val();
+    const dobDay = this.ui.dobDay.val();
+    const dobYear = this.ui.dobYear.val();
+
+    if (!dobMonth || !dobDay || !dobYear) {
+      this.model.set({ birth_date: '' });
+      return;
+    }
```

Comment:
This switches the `Submit` button to disabled if the birth date is no longer valid.